pyquil/quil.py

```diff
@@ -73,6 +74,20 @@ def __iter__(self):
         """
         return self.actions.__iter__()
 
+    def get_qubits(self):
```

Comment:
I agree, I think it makes sense to count measurements even if they happen on qubits where no gates have been applied. A use case would be to test something like crosstalk. Will add that